### PR TITLE
Set AspNetCoreHostingModel in csproj templates

### PR DESCRIPTION
**Pull request by pakrym**
I didn't change F# templates or razor class library template.

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/StarterWeb-FSharp.fsproj.in [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
     <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
     <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
+    <AspNetCoreHostingModel>inprocess</AspNetCoreHostingModel>
```
Comment (shirhatti) [inline src/Microsoft.DotNet.Web.ProjectTemplates/StarterWeb-FSharp.fsproj.in]: Looks like you did change an FSharp project

**Comment by shirhatti** [comment]
Why did you exclude FSharp projects?

**Comment by Eilon** [comment]
Right, template changes should include all applicable project types.

**Comment by Eilon** [comment]
@pakrym thanks, you are now our resident F# expert :trollface:

**Pull request merged**